snippets/perl/tokendump.pl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env perl
```

Comment:
Suggest moving this under tools/perl/tokendump.pl since this a tool/utility and isn't demonstrating any APIs.

Reply:
I assume you mean instead of snippets/perl/, move it tools/perl/tokendump?  What is the .md extension?